modules/cli-logging-in.adoc

```diff
@@ -12,6 +12,12 @@ You can log in to the `oc` CLI to access and manage your cluster.
 * You must have access to an {product-title} cluster.
 * You must have installed the CLI.
 
+[NOTE]
+====
+If you want to access a cluster in the internet that you can only access over an HTTP proxy server, then you can set the `HTTP_PROXY` and `HTTPS_PROXY` variables.
+These environment variables are respected by `oc` so that all communication with the cluster will go through the HTTP proxy.
```

Comment:
* "by the `oc` CLI"
* "so that all communication with the cluster goes through"